adapters/visx/visx.go

```diff
@@ -14,11 +14,38 @@ type VisxAdapter struct {
 	endpoint string
 }
 
+type visxBid struct {
```

Comment:
I'm curious. What was broken with the `bid.ID` and `bid.CrID` previously?

Reply:
Problems with `bid.CrID` was resolved on the server side.

About `bid.ID` – our ad server doesn't return this field in the ad response. Since this parameter is required we set it in s2s adapter.